libs/platform/user/ebpf_platform_user.cpp

```diff
@@ -263,6 +198,20 @@ ebpf_interlocked_compare_exchange_int32(_Inout_ volatile int32_t* destination, i
     return InterlockedCompareExchange((long volatile*)destination, exchange, comperand);
 }
 
+void*
+ebpf_interlocked_compare_exchange_pointer(_Inout_ void* volatile* destination, void* exchange, void* comperand)
+{
+    return InterlockedCompareExchangePointer((void* volatile*)destination, exchange, comperand);
+}
+
+uint32_t
+ebpf_random_uint32()
```

Comment:
There's a helper we need (issue #342) called
`u32 bpf_get_prandom_u32(void)`.
Should this be renamed, or should that helper just call this API?